var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -573,7 +573,7 @@ def cmake_args(self):
 
         if "+all_targets" not in spec:  # all is default on cmake
 
-            targets = ["NVPTX", "AMDGPU"]
+            targets = ["NVPTX", "AMDGPU", "WebAssembly", "BPF"]
```

Comment:
xD

Reply:
Yeah, I was thinking about that. I also suspect that for now no-one cares about WASM and BPF, but people might want to turn on VectorEngine support, which is coming to a LLVM 14 near you.  cc: @efocht